<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CConditionEditDialog</class>
 <widget class="QDialog" name="CConditionEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>状态条件</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButton_timeCondition">
        <property name="text">
         <string>时间</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_time">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QRadioButton" name="radioButton_after">
             <property name="text">
              <string>之后</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_before">
             <property name="text">
              <string>之前</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QTimeEdit" name="timeEdit_time">
             <property name="displayFormat">
              <string>HH:mm</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_idleCondition">
        <property name="text">
         <string>计算机空闲</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_idle">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QTimeEdit" name="timeEdit_idle">
             <property name="dateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>1</second>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="minimumTime">
              <time>
               <hour>0</hour>
               <minute>0</minute>
               <second>1</second>
              </time>
             </property>
             <property name="displayFormat">
              <string>HH:mm:ss</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_fileCondition">
        <property name="text">
         <string>文件存在</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_fileTrigger">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLineEdit" name="lineEdit_watchFile">
           <property name="placeholderText">
            <string>监视文件路径</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_explorer">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../qrc.qrc">
             <normaloff>:/image/folder.png</normaloff>:/image/folder.png</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_processCondition">
        <property name="text">
         <string>进程运行/退出</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_processTrigger">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QRadioButton" name="radioButton_processRunning">
             <property name="text">
              <string>进程运行</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_processNotRunning">
             <property name="text">
              <string>进程退出</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLineEdit" name="lineEdit_watchProcess">
             <property name="placeholderText">
              <string>监视进程名</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_selectProcess">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../qrc.qrc">
               <normaloff>:/image/search.png</normaloff>:/image/search.png</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_error">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>160</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qrc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CConditionEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_timeCondition</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_time</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_fileCondition</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_fileTrigger</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_processCondition</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_processTrigger</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_idleCondition</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_idle</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
